src/ndlbulkdownload/args.py

```diff
@@ -50,6 +50,13 @@ def arg_parser():
                         action='append',
                         help='Add query param key/value pair')
 
+    parser.add_argument('--output_dir',
+                        metavar='O',
+                        type=str,
+                        default=os.getcwd(),
+                        help="Directory to output files. Example: /tmp/ndl (default: current directory)."
+                             " Directory, if do not exists, will be created")
```

Comment:
The help text is currently failing lint.  I need to setup the project to help catch these before people submit.  I also need to spend some time fixing up the configuration of lint rules, as well as reorganizing some of the help variables as there's not a lot of real-estate due to the current indentation level.